thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/DetectionJobResource.java

```diff
@@ -217,73 +218,123 @@ public Response computeSeverity(@NotNull @QueryParam("collection") String collec
    * Regenerates anomalies for each window separately
    *
    * As the anomaly result regeneration is a heavy job, we move the function from Dashboard to worker
-   * @param id anomaly function id
+   * @param ids a string containing multiple anomaly function ids, separated by comma (e.g. f1,f2,f3)
```

Comment:
Maybe we can use function/{id}/replay for single function,
and create another one function/replay?ids="f1,f2,f3" for multiple functions replay at a batch if you really think it is truly necessary.